### PR TITLE
Add remote API for cryocell indexing

### PR DESCRIPTION
**Pull request by jnwatson**

### CODE REVIEW

File: synapse/cryotank.py [hunk 1]
```diff
@@ -732,28 +923,28 @@ def iidFromProp(self, prop):
...
         '''
         Add an index to the cryotank
 
         Args:
             prop (str):  the name of the property this will be stored as in the normalized record
             syntype (str):  the synapse type this will be interpreted as
-            datapath (str):  the datapath spec against which the raw record is run to extract a single field
-        that is passed to the type normalizer.
-            *args (str):  additional datapaths that will be tried in order if the first isn't present.
+            datapaths (str):  datapaths that will be tried in order.
```
Comment (invisig0th) [inline synapse/cryotank.py]: not a (str)

File: synapse/cryotank.py [hunk 2]
```diff
@@ -732,28 +923,28 @@ def iidFromProp(self, prop):
...
         Returns:
             None
         Note:
-            Additional datapaths will be tried iff prior datapaths are not present, and *not* if
+            Additional datapaths will be tried if and only if prior datapaths are not present, and *not* if
```
Comment (invisig0th) [inline synapse/cryotank.py]: lets replace "will be tried if and only if" with the simpler "will only be tried if"

File: synapse/cryotank.py (hunk 1 above)
Comment (jnwatson) [reply synapse/cryotank.py]: ok

File: synapse/cryotank.py (hunk 2 above)
Comment (jnwatson) [reply synapse/cryotank.py]: ok

**Pull request merged**